src/types.ts

```diff
@@ -309,8 +310,8 @@ export type SerializedAXNode = {
   required?: boolean,
   selected?: boolean,
 
-  checked?: boolean | 'mixed',
-  pressed?: boolean | 'mixed',
+  checked?: 'checked' | 'unchecked' | 'mixed',
```

Comment:
nit: It might be easier to do 'true' | 'false' | 'mixed' to match the aria spec here. But I guess we already have it in RPC.

Reply:
We can follow up with changing the rpc. Let me put it into the list.